conn.go

```diff
@@ -607,15 +606,16 @@ func (c *Client) pingLoop(ctx context.Context, errs chan error, wg *sync.WaitGro
 			if time.Since(c.conn.lastPong) > c.Config.PingDelay+(60*time.Second) {
 				// It's 60 seconds over what out ping delay is, connection
 				// has probably dropped.
-				errs <- ErrTimedOut{
```

Comment:
Holy shit. I wonder if this error channel was half the reason I was ever having problems with spawning so many clients and having them time out. I never noticed it. That channel if backed up would have done exactly that.

I'll still reap a lot of benefits from my changes for my use-case - but still - I may never have been pushed to making it if my theory is true.